docs/USAGE-GUIDE.md

```diff
@@ -383,6 +384,7 @@ failure. When an s2n function returns a failure, s2n_errno will be set to a valu
 corresponding to the error. This error value can be translated into a string
 explaining the error in English by calling s2n_strerror(s2n_errno, "EN");
 A string containing internal debug information, including filename and line number, can be generated with `s2n_strerror_debug`
+error name can be obtained by calling s2n_strerror_name
```

Comment:
nit: put `s2n_strerror_name` into ` as done above for `s2n_strerror_debug`

Reply:
Sure.